src/index.ts

```diff
@@ -59,14 +59,14 @@ export default function useSound(
     };
   });
 
-  // When the URL changes, we have to do a whole thing where we recreate
+  // When the `src` changes, we have to do a whole thing where we recreate
   // the Howl instance. This is because Howler doesn't expose a way to
   // tweak the sound
   React.useEffect(() => {
     if (HowlConstructor.current && sound) {
       setSound(
         new HowlConstructor.current({
-          src: [url],
+          src: Array.isArray(src) ? src : [src],
```

Comment:
Instead of using this `Array.isArray(src) ? src : [src]` logic in multiple places within this PR, it might be good to DRY (don't repeat yourself) and do it once at the top of the hook, then reuse that value everywhere else?